opacus/privacy_engine.py

```diff
@@ -80,6 +79,92 @@ def __init__(self, secure_mode=False):
                 "one last time before production with ``secure_mode`` turned on."
             )
 
+    def _prepare_accountant(self, mechanism: str) -> IAccountant:
```

Comment:
Should we move that to accountants/__init__.py? In any case it is a "static" method.

Reply:
Hmm, that is an excellent point